ProcessLib/LiquidFlow/LiquidFlowProcess.h

```diff
@@ -88,7 +101,7 @@ class LiquidFlowProcess final : public Process
 
     const int _gravitational_axis_id;
     const double _gravitational_acceleration;
-    LiquidFlowMaterialProperties _material_properties;
+    const std::unique_ptr<LiquidFlowMaterialProperties> _material_properties;
```

Comment:
Why not a `const reference`? If it would be a `const&` maybe the implementation of `getFluidProperties()` above can return a `const& FluidProperties` and put it outside of the class into a `std::vector`?